PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+# Purpose
+
+<Write your PR's purpose here>
+
+# API changes
+
+<Write the functions and structures you added or removed, if any>
+
+# PR checklist
+Write an x in the below boxes if they are true.
+- [ ] I agree to dual-license my contribution under the MIT license and the Apache License version 2.0
+- [ ] I've tested my code on at least one supported platform
+- [ ] I ran rustfmt on my code
+- [ ] There were breaking changes in my PR so I:
+    - [ ] Updated the book to reflect these changes.
+    - [ ] Updated the examples to use these changes correctly.
+    - [ ] Updated the corresponding doc comments if necessary.
+- [ ] I am providing an optimization to amethyst, and I have benchmarks proving it is an improvement.
```

Comment:
I really think this is the job of the reviewers. I mean why would you merge an optimization which either isn't proven to be one or isn't obvious. E.g. there's no need to write a benchmark if a call to `.clone()` has been eliminated, but if somebody rewrites an algorithm, it's just common sense to ask for a benchmark, isn't it?

Reply:
Fair point.